test/README

```diff
@@ -219,3 +219,32 @@ Still, within a long test, try to have independent sections, where
 each section returns the machine to more or less the state that it was
 in before the section.  This makes it easier to run these sections
 ad-hoc when doing incremental development.
+
+## Running Selenium tests locally
+
+Selenium tests run as a part of GiHub PR CI.
+To run selenium tests locally you will need to do steps described below.
+
+Make sure, first of all, to run:
+
+./bots/image-prepare
+
+By default you'll get corresponding image for system you're running currently.
+E.g. if you run Fedora-27, you'll get Fedora-27 image.
```

Comment:
Selenium/avocado tests (currently) run on fedora-26. Other images don't have selenium and avocado installed.